packages/contracts/test/voting/token-voting.ts

```diff
@@ -172,23 +205,23 @@ describe('TokenVoting', function () {
         )
       )
         .to.emit(voting, VOTING_EVENTS.PROPOSAL_CREATED)
-        .withArgs(id, ownerAddress, dummyMetadata);
+        .withArgs(id, signers[0].address, dummyMetadata);
 
       const block = await ethers.provider.getBlock('latest');
 
       const vote = await voting.getProposal(id);
       expect(vote.open).to.equal(true);
       expect(vote.executed).to.equal(false);
-      expect(vote._relativeSupportThresholdPct).to.equal(2);
-      expect(vote._totalSupportThresholdPct).to.equal(1);
+      expect(vote._supportThreshold).to.equal(supportThreshold);
+      expect(vote._participationThreshold).to.equal(participationThreshold);
       expect(vote.snapshotBlock).to.equal(block.number - 1);
       expect(vote.totalVotingPower).to.equal(1);
       expect(vote.yes).to.equal(0);
       expect(vote.no).to.equal(0);
 
       expect(vote.startDate.add(minDuration)).to.equal(vote.endDate);
 
-      expect(await voting.canVote(1, ownerAddress)).to.equal(false);
+      expect(await voting.canVote(1, signers[0].address)).to.equal(false);
```

Comment:
Why removing ownerAddress? low importance

Reply:
I wanted to have less variables and preferred to write the number explicitly - especially for AddresslistVoting, it made more sense to use the numbers as we write `signers[1]`, `signers[2]`, ... all the time and then it is more unclear if we suddenly use `ownerAddress` instead of `signers[0]`.

Reply:
I personally prefer reading tests in terms of `if (balanceOf(owner) >= balanceOf(delegate)) { owner.transfer(delegate, 1eth)}`  as opposed of indexes.

This is going to be helpful for the future "ourselves" and our chances of catching blind spots more easily